src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/recordcomponentname/Example2.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RecordComponentName">
+        <property name="format" value="^[a-z]+$"/>
+    </module>
+  </module>
+</module>
+*/
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.recordcomponentname;
+
+// xdoc section -- start
+class Example2 {
+  record MyRecord1(String value, int other) {}
+
+  record MyRecord2(String... strings) {}
```

Comment:
Please keep examples the same.
This record is absent in example1.

To spot it yourself , please generate website, and copy content of examples to some diff text tool, there are few free in web.
Looks at issue description images for example.

Reply:
Done